export const authErrors = {
    authFail: new Error("auth failed"),
    alreadyExists: new Error("this user already exists")
}